meetings/2022-03-31.md

```diff
@@ -0,0 +1,127 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2022-03-31
+
+## Links
+
+* **Recording**:  <http://www.youtube.com/watch?v=DFlvxeDa_nQ>
+* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1196>
+
+## Present
+
+*Antoine du Hamel @aduh95 (TSC)
+*Anatoli Papirovski @apapirovski (TSC)
+*Beth Griggs @BethGriggs (TSC)
+*Colin Ihrig @cjihrig (TSC)
+*Gireesh Punathil @gireeshpunathil (TSC)
+*Matteo Collina @mcollina (TSC)
+*Michael Dawson @mhdawson (TSC)
+*Darshan Sen @RaisinTen (TSC)
+*Richard Lau @richardlau (TSC)
+*Robert Nagy @ronag (TSC)
+*Tobias Nießen @tniessen (TSC)
+*Rich Trott @Trott (TSC)
+
+## Agenda
+
+### Announcements
+
+* Richard, release proposal for final 12.x release. Also reminder that it goes EOL
+  at the end of April so make sure you are planning to move off.
+* Rich, 18.x coming out in a few weeks as well.
+
+### CPC and Board Meeting Updates
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* CPC update - no updates this week. Elections are coming up soon.
+* Board meeting - Nothing of note this week.
+
+### nodejs/node
+
+* export v8 trace API [#42462](https://github.com/nodejs/node/pull/42462)
+  * James asked that we defer this to a meeting that he can make.
```

Comment:
Sorry, I got confused with this one and the "Publishing specs for `Buffer` and `EventEmitter`" issue, which is the one James asked to defer, not this one.